zingocli/tests/darksidewalletd_tests.rs

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I like the new test file.  These are a fundamentally new type of test..   why not give them their own file?

Reply:
I'm not done with that. Tests wouldn't compile for some reason that is probably obvious for a real Rust dev but not to me 😅